vcr/store_test.go

```diff
@@ -52,9 +52,9 @@ func TestVcr_StoreCredential(t *testing.T) {
 	t.Run("ok", func(t *testing.T) {
 		ctx := newMockContext(t)
 
-		ctx.tx.EXPECT().Subscribe(gomock.Any(), gomock.Any()).Times(2)
 		ctx.vcr.Configure(core.ServerConfig{Datadir: io.TestDirectory(t)})
 		ctx.keyResolver.EXPECT().ResolveSigningKey(gomock.Any(), nil).Return(pk, nil)
+		ctx.docResolver.EXPECT().Resolve(gomock.Any(), gomock.Any()).Return(nil, nil, nil)
```

Comment:
Why do we use `gomock.Any()` here? We do want to verify the mocked methods are called with the actual paremeters right? Unless it's something like a `Context` I wouldn't use `gomock.Any()`

Reply:
removed lots of gomock.Any